README.md

```diff
@@ -129,7 +138,20 @@ Let's say the Posts in the previous question are available at `http://localhost:
     3. If Step 1 is unsuccessful, print an error message to the console  
 
 ```js
-// Your answer goes here...
+
+$(document).ready(()=> {
+
+  $.ajax({
+    type: "GET",
+    dataType: 'json',
+    url: "/posts"  
+  }).done((response) => {
+    console.log(response);
+  }).fail((response) => {
+    console.log("Request failed");
+  })
+}
+
 ```
 
```

Comment:
👌 👌 👌

Since you aren't doing anything with the DOM, then you don't need `+$(document).ready`